net.trowell.typesetter.json

```
@@ -0,0 +1,110 @@
+{
+    "id" : "net.trowell.typesetter",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "49",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "typesetter",
+    "finish-args" : [
+        "--device=dri",
+        "--share=network",
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--share=ipc"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin",
+        "env" : {
+            "RUST_BACKTRACE" : "1",
+            "RUST_LOG" : "typesetter=debug"
+        }
+    },
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name": "typst-mime",
+            "buildsystem": "simple",
+            "build-commands": [
+                "mkdir -p /app/share/mime/packages",
+                "install -Dm644 typst.xml /app/share/mime/packages/typst.xml",
+                "update-mime-database /app/share/mime"
```

```suggestion
                "install -Dm644 typst.xml /app/share/mime/packages/$FLATPAK_ID.xml",
```

but I don't see why this is not installed by your build system. Regardless of https://github.com/flathub/flathub/pull/7133/files#r2505708383